docs/content/docs/user-docs/field-export.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Export a Field"
+description: "Using the FieldExport custom resource"
+lead: "Exporting a spec or status field using the FieldExport resource"
+draft: false
+menu:
+  docs:
+    parent: "getting-started"
+weight: 65
+toc: true
+---
+
+ACK controllers are intended to manage your AWS infrastructure using Kubernetes
+custom resources. Their responsibilities end after managing the lifecycle of
+your AWS resource and do not extend into binding to applications running in the
+Kubernetes data plane. The ACK `FieldExport` custom resource was designed to
+bridge the gap between managing the control plane of your ACK resources and
+using the properties of those in your application.
+
+The `FieldExport` resource configures any of the ACK
+controllers to bind and export any spec or status field from an ACK resource
+into a Kubernetes config map or secret. These fields are automatically updated
+as the field value changes over time. You are then able to mount the config map
+or secret onto your Kubernetes pods as environment variables to ingest those
+values. 
+
+`FieldExport` is included by default in every ACK controller installation and
+can be used to reference any field within the `Spec` or `Status` of any ACK
+resource.
+
+## Using a FieldExport
+
+The `Spec` and `Status` fields of the `FieldExport` custom resource definition
+are available in the [API reference][spec-reference]. For this example, we will
+be creating an [S3 Bucket][bucket-spec] and exporting the `Status.Location`
+field into a `ConfigMap`.
+
+```yaml
+apiVersion: s3.services.k8s.aws/v1alpha1
+kind: Bucket
+metadata:
+  name: application-user-data
+spec:
+  name: doc-example-bucket
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: application-user-data-cm
+data: {}
+---
+apiVersion: services.k8s.aws/v1alpha1
+kind: FieldExport
+metadata:
+  name: export-user-data-bucket
+spec:  
+  to:
+    name: application-user-data-cm # Matches the ConfigMap we created above
+    kind: configmap
+  from:
+    path: ".status.location"
+    resource:
+      group: s3.services.k8s.aws
+      kind: Bucket
+      name: application-user-data
+```
```

Comment:
🤔 It may be good idea to demonstrate how to export two fields. This is the first question I have in my mind after reading this, and it answer the "What does it look like when I export more than one field?" question.